Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -62,6 +62,22 @@ public override string ToString()
             return "<Motorcycle>{ Brand: " + Brand + ", color: " + color + ", canOffroad: " + canOffroad + " }";
         }
     }
+
+    public class MOTORCYCLE : Vehicle
```

Comment:
Why are these capitalized, if they're for testing purposes please put a comment as to that effect.

Reply:
They are indeed for testing purposes and comments explaining that are now added.

Reply:
The Type names are not affected by this casing property. You should not need to do this.

Reply:
We could have named this class anything but it made sense to just name it capitalized MOTORCYLE because it's the all-caps version of the Motorcycle class.